transport/build.gradle.kts

```diff
@@ -148,6 +148,16 @@ tasks {
         archiveClassifier.set("javadoc")
         from("./deployment")
     }
+
+    register("generateGenesysCloudMessengerTransportPodspec") {
+        doLast {
```

Comment:
Really minor. 
It looks like this task belongs to group: `group = "publishing"`

Reply:
> Really minor.
> It looks like this task belongs to group: `group = "publishing"`

I'll add that group to the Task. While I'm at it, I'll add a description for the Task too.